students/Sally_Shen/lesson4/mailroom2.py

```diff
@@ -0,0 +1,123 @@
+
+
+'''choms'''
+
+import sys
+import os
+
+print("in mailroom")
+'''Writing first name is easier for testing.'''
+donors ={"Fred Gates": [100.90, 200, 50], "Jeff Bezos": [100, 50, 600], "Mark Zuckerberg": [200, 50, 500],
+         "Paul Allen": [300, 40, 200]}
+
+#donors = {{"first_name": "Fred", "last_name": "Gates"}: [[100.90, 200, 50], 265.98]}
+
+
+def thank_you():
+    response = input("Type your donor name, if don't know, type list to pick")
+    '''If the user types list show them a list of the donor names and re-prompt'''
+    if response == "list":
+        for key in donors.keys():
+            print(key)
+        response = input("Please type your donor name.")
+
+    if response not in donors.keys():
+        donors[response] = []
+
+    newDonation = input("how much to donate?")
+    donors[response].append(float(newDonation))
+    email = '''
+    Dear {0},
+    Thank you for donating {1}!\n'''.format(response, newDonation)
+    return email
```

Comment:
After returning email here, you are not printing it. You should be printing a thank you note after each individual donation.